### PR TITLE
fix: the app to allow a call to the POST /remittance endpoint

### PR DESCRIPTION
**Pull request by snackycracky**

### CODE REVIEW

**Comment by kyhsa93** [comment]
@snackycracky can you any description for this pr plz?

File: src/account/infrastructure/repository.ts [hunk 1]
```diff
@@ -6,9 +6,10 @@ import AccountEntity from '@src/account/infrastructure/entity/account';
 import Account from '@src/account/domain/model/account';
 import AccountRepository from '@src/account/domain/repository';
 import AccountFactory from '@src/account/domain/factory';
+import { Inject } from '@nestjs/common';
 
 export default class AccountRepositoryImplement implements AccountRepository {
-  constructor(private readonly accountFactory: AccountFactory) {}
+  constructor(@Inject('AccountFactory') private readonly accountFactory: AccountFactory) {}
```
Comment (kyhsa93) [inline src/account/infrastructure/repository.ts:12]: there is no change needed.
if injected dependency type and injected implement name are same, `@Inject()` is not required.

File: src/account/domain/factory.ts [hunk 2]
```diff
@@ -3,7 +3,10 @@ import bcrypt from 'bcrypt';
 import AccountCreated from '@src/account/domain/event/account.opened';
 import Account, { AccountAttributes } from '@src/account/domain/model/account';
 import Password from '@src/account/domain/model/password';
+import { Injectable } from '@nestjs/common';
 
+
+@Injectable()
```
Comment (kyhsa93) [inline src/account/domain/factory.ts:9]: `@Injectable()` is not required.

File: src/main.ts [hunk 3]
```diff
@@ -9,7 +9,7 @@ function throwError(error: Error): never {
 }
 
 async function bootstrap(): Promise<void> {
-  const app = await NestFactory.create(ApplicationModule, { cors: true });
+  const app = await NestFactory.create(ApplicationModule, { cors: true, logger: ['debug', 'verbose','warn','error','log'] });
```
Comment (kyhsa93) [inline src/main.ts:12]: i think logger option is not needed in this app.
how about you?
and i want to your opinion about use all logger level.

**Review by kyhsa93 — COMMENTED** [review]
thank you for create this pull request.
i created some review comments. 
please check my comments, and if you have some question or opinion write with comments.
thank you.

**Comment by snackycracky** [comment]
Is the POST /remittance endpoint working on the master branch ? For me it has several issues.

**Comment by snackycracky** [comment]
if the master branch is fine then you can close this pr.

vG:Nils

**Comment by kyhsa93** [comment]
this PR has some conflict, so i close this PR
thank you

**Pull request closed without merging**